velox/common/time/Timer.h

```diff
@@ -43,4 +45,20 @@ size_t getCurrentTimeMs();
 // Returns the current epoch time in microseconds.
 size_t getCurrentTimeMicro();
 
+// Match the input time in nanoseconds to the most appropriate unit and return a
+// string value. Possible units are nanoseconds(ns), microseconds(us),
+// milliseconds(ms), seconds(s). The default precision is 4 decimal digits.
+static std::string prettyPrintTimeInNanos(uint64_t time, int precision = 4) {
```

Comment:
Let's move the body of the function to .cpp (to allow for faster builds). 

I wonder if a shorter name prettyPrintNanos would be better. I'm also seeing this used for milliseconds, perhaps, add prettyPrintMillis as well? succinctNanos / succinctMillis are other names to consider. We'd want succinctBytes as well.

A test would be nice as it would allow us to make sure the numbers are printed correctly for all the use cases.